Modules/Scripted/DataProbe/DataProbe.py

```diff
@@ -331,6 +294,52 @@ def _roundInt(value):
     else:
       self.frame.parent().text = "Data Probe"
 
+
+  def UpdateCursorPositionDisplayText(self, xyz, ras, sliceNode, sliceLogic):
+    spacing = "%.1f" % sliceLogic.GetLowestVolumeSliceSpacing()[2]
+    if sliceNode.GetSliceSpacingMode() == slicer.vtkMRMLSliceNode.PrescribedSliceSpacingMode:
+      spacing = "(%s)" % spacing
+
+    self.viewInfo.text = \
+      "  {layoutName: <8s}  RAS: ({ras_x:6.1f}, {ras_y:6.1f}, {ras_z:6.1f})  {orient: >8s} Sp: {spacing:s}" \
+      .format(layoutName=sliceNode.GetLayoutName(),
+              ras_x=ras[0],
+              ras_y=ras[1],
+              ras_z=ras[2],
+              orient=sliceNode.GetOrientationString(),
+              spacing=spacing
+              )
+
+    def _roundInt(value):
+      try:
+        return int(round(value))
+      except ValueError:
+        return 0
+
+    hasVolume = True
```

Comment:
@Punzo  I am integrating the changes. Note that I will keep this as `hasVolume = False` to match the existing behaviour.